IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -199,4 +204,14 @@ - (BOOL)shouldRefreshWithInterval:(NSUInteger)refreshInterval
     return shouldRefresh;
 }
 
+- (NSUInteger)refreshInterval
+{
+    if (self.expiryInterval > 0)
+    {
+        return self.expiryInterval / 10;
```

Comment:
If PRT expires in 10 days, we will refresh PRT everyday. Did i get it right?

Reply:
If we use it every day, yes. Chances are high that broker won't be called every day. That is why trying to be a little aggressive here. If we see it being too often, we can reduce frequency.

Reply:
I took a look at the data, and most users use PRT 1-3 times in a 14 day window. So I think this frequency actually needs to be increased to make sure PRT is kept fresh. I was thinking to set it to 20 which means that PRT refresh will be attempted every 16 hours. @jasoncoolmax, what do you think?

Reply:
Discussed offline. We'll keep it at 10 until we have telemetry that we need to be more aggressive :)